presto-orc/src/main/java/com/facebook/presto/orc/reader/TimestampBatchStreamReader.java

```diff
@@ -192,7 +192,7 @@ private void openRowGroup()
     }
 
     @Override
-    public void startStripe(InputStreamSources dictionaryStreamSources, Map<Integer, ColumnEncoding> encoding)
+    public void startStripe(LongDictionaryProvider longDictionaryProvider, InputStreamSources dictionaryStreamSources, Map<Integer, ColumnEncoding> encoding)
```

Comment:
Conceptually, TimestampBatchStreamReader taking in LongDictionaryProvider does not make sense.

Reply:
I agree with passing in the Stripe in this case. However, regarding whether passing in a dictionary provider makes sense, the current code base already passes dictionaryStreamSources into all non-dictionary-encoding column readers anyway. :D In the future I plan to replace dictionaryStreamSources entirely, with support also for string dictionaries streams. I currently named the current class LongDictionaryProvider just to be clear what it supports.